packages/analytics-browser/src/browser-client.ts

```diff
@@ -18,8 +17,10 @@ import { Context } from './plugins/context';
 import { useBrowserConfig, createTransport, createFlexibleStorage } from './config';
 import { parseOldCookies } from './cookie-migration';
 
+export type IAmplitudeBrowser = AmplitudeBrowser;
```

Comment:
probably do not need to expose if `AmplitudeBrowser` is private?

Reply:
I will remove it from this PR.